go-runtime/ftl/ftltest/ftltest.go

```diff
@@ -73,10 +74,11 @@ func WithProjectFiles(paths ...string) Option {
 	var preprocessingErr error
 	if len(paths) == 0 {
 		envValue, ok := os.LookupEnv("FTL_CONFIG")
-		if !ok {
-			preprocessingErr = fmt.Errorf("loading project files: no path provided and FTL_CONFIG environment variable not set")
+		if ok {
+			paths = strings.Split(envValue, ",")
+		} else {
+			paths = pc.ConfigPaths(paths)
```

Comment:
Should we confirm that `paths` isn't [empty](https://github.com/TBD54566975/ftl/blob/main/common/projectconfig/projectconfig.go#L50) and return the same `preprocessingErr` error from before?

Reply:
My intention here is to have the [same behavior](https://github.com/TBD54566975/ftl/blob/cb3b3703ae24052f51794f3c7d5aa908406f9c86/common/projectconfig/merge.go#L15) as `main.go`, which allows `LoadConfig` to return an empty config after attempting to fall back on the envar, then the gitroot. I think having that parity makes sense?

Reply:
Ah, I see. I was wondering because I can't think of a reason why someone would use `WithProjectFiles` without any paths. From that perspective, I'd find it useful to get the error message during development so that I know the way I'm using `WithProjectFiles` isn't actually doing anything. `main.go` wouldn't want to error in that case because the project toml isn't required, whereas if someone is explicitly invoking `WithProjectFiles`, I'd assume they have some intent to actually use a project toml.

Reply:
I think it's clear that if you're not passing explicit config files to `WithProjectFiles()` that whatever its described behaviour is is in effect. The `ftl-project.toml` file is designed to be the "default" for a project, and having it consistent across the CLI and tests is fine.